app/code/community/Bolt/Boltpay/controllers/ShippingController.php

```diff
@@ -78,18 +78,15 @@ public function indexAction()
                 'telephone' => $shipping_address->phone
             );
 
-            $display_id = $request_data->cart->display_id;
-
+            $quote_id = $request_data->cart->order_reference;
```

Comment:
nit: going forward let's use camelCase within codebase for consistency (and to follow magento's convention).